daphne/src/roles/leader.rs

```diff
@@ -439,8 +443,13 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             .try_put_agg_share_span(task_id, task_config, agg_span)
```

Comment:
Update the comment above ("if we do detect an error (at least one report was replayed, the span overlaps with a collected batch, etc.")